nix/pkgs/iree-runtime.nix

```diff
@@ -0,0 +1,224 @@
+{
+  lib,
+  cmake,
+  ninja,
+  rv32-stdenv,
+  iree,
+  fetchFromGitHub,
+}:
+let
+  version = "3.4.0rc20250331";
+  iree-googletest-version = "c8393f8554419dc27b688c535b8fa4afb82146a4";
+  iree-googletest-src = fetchFromGitHub {
+    owner = "google";
+    repo = "googletest";
+    rev = iree-googletest-version;
+    hash = "sha256-K48kVGMyoWWcDHXhHM98NORZerBm6wbnYf50GkK4B2Y=";
+  };
+  iree-flatcc-version = "9362cd00f0007d8cbee7bff86e90fb4b6b227ff3";
+  iree-flatcc-src = fetchFromGitHub {
+    owner = "dvidelabs";
+    repo = "flatcc";
+    rev = iree-flatcc-version;
+    hash = "sha256-umZ9TvNYDZtF/mNwQUGuhAGve0kPw7uXkaaQX0EzkBY=";
+  };
+in
+rv32-stdenv.mkDerivation {
+  pname = "iree";
```

Comment:
Please rename this to `iree-runtime`. Using package name correspond to nix filename could help other debug nix expression.